pkg/api/aim/controller/runs.go

```diff
@@ -62,6 +63,26 @@ func (c Controller) GetRunMetrics(ctx *fiber.Ctx) error {
 	return ctx.JSON(resp)
 }
 
+// CreateRunArtifact handles `POST /runs/:id/artifacts` endpoint.
+func (c Controller) CreateRunArtifact(ctx *fiber.Ctx) error {
```

Comment:
Should this be in the `mlflow` api space, along with `log_metric` etc? The `mlflow` client has `log_artifact` for this.